test/e2e/functional/tests/e2e.go

```diff
@@ -1126,29 +1137,20 @@ var _ = Describe("metallb", func() {
 			err = testclient.Client.ConfigMaps(OperatorNameSpace).Delete(context.Background(), consts.MetalLBConfigMapName, metav1.DeleteOptions{})
 			Expect(err).ToNot(HaveOccurred())
 
-			Eventually(func() (string, error) {
+			Eventually(func() bool {
 				var err error
-				configMap, err = testclient.Client.ConfigMaps(OperatorNameSpace).Get(context.Background(), consts.MetalLBConfigMapName, metav1.GetOptions{})
-				if err != nil {
-					return "", err
-				}
-				return configMap.Data[consts.MetalLBConfigMapName], err
-			}, metallbutils.Timeout, metallbutils.Interval).Should(MatchYAML(`address-pools:
-- name: addresspool1
-  protocol: layer2
-  addresses:
-  - 1.1.1.1-1.1.1.100
-`))
+				_, err = testclient.Client.ConfigMaps(OperatorNameSpace).Get(context.Background(), consts.MetalLBConfigMapName, metav1.GetOptions{})
```

Comment:
Why this change? The previous test was looking after a particular rendered value.

Reply:
same as above.